src/03-control-flow.js

```diff
@@ -128,7 +167,17 @@ function aroundTheWorldGreetingWithSwitch(language) {}
  *  calculateLetterGrade(99);
  *  //> "A"
  */
-function calculateLetterGrade(grade) {}
+function calculateLetterGrade(grade) {
+  if (grade >= 90) {
+    return 'A'
+  } else if (grade >= 80) {
+    return 'B'
+  } else if (grade >= 70) {
+    return 'C'
+  } else if (grade >= 60) {
+    return 'D'
+  } else {return 'F'}
```

Comment:
This is so easy to read Victor.  The only thing i would say is lets move our final return down one line and add a  ` ; ` after our returns